NEWS.md

```diff
@@ -1,3 +1,10 @@
+## Changes since the last version
+
+- `score_forecasts()` is now implemented.
+
+  Minimally one should have the `forecasts` dataframe produced by `load_forecasts()` and the truth dataframe produced by `load_truth()` to calculate scores. If one desires to specify a subset of all available scores, one should consult [this reference](https://epiforecasts.io/scoringutils/reference/eval_forecasts.html#details) for valid scores in the `desired_score_types` vector.
```

Comment:
minor comment, but it's not clear to me what this paragraph is for. should this be in the documentation of the function rather than in news?